### PR TITLE
[update] rename 'label' as 'type'

### PR DESCRIPTION
**Pull request by Elssky**
https://github.com/apache/incubator-graphar/issues/604

### CODE REVIEW

**Comment by Elssky** [comment]
> [apache/incubator-graphar#604](https://github.com/apache/incubator-graphar/issues/604)

Rename all `label ` in yml to `type`
Rename all `src_label ` in yml to `src_type`
Rename all `dst_label ` in yml to `dst_type`
Rename all `edge_label` in yml to `edge_type`

File: .asf.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ github:
   description: >-
     Auxiliary testing data files for Apache GraphAr (Incubating).
   homepage: https://graphar.apache.org/
-  labels:
```
Comment (acezen) [inline .asf.yaml]: this is not the testing file of graphar, it's the configuration of asf repo, so the `labels` is not what we want to change

Comment (Elssky) [reply .asf.yaml]: yes, fix it

**Review by acezen — APPROVED** [review]
LGTM

**Pull request merged**